namespaces/live-1.cloud-platform.service.justice.gov.uk/peoplefinder-development/resources/elasticsearch.tf

```diff
@@ -15,5 +15,5 @@ module "peoplefinder_es" {
   team_name              = "peoplefinder"
   elasticsearch-domain   = "es"
   namespace              = "peoplefinder-development"
-  elasticsearch_version  = "1.5"
+  elasticsearch_version  = "6.8"
```

Comment:
@Polsonby can u also update the module release to use 3.0 which supports modern versiion
`source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticsearch?ref=3.0"`

Reply:
@poornima-moj Thanks, have updated it :)